browser/ui/webui/settings/brave_privacy_handler.cc

```diff
@@ -96,6 +101,12 @@ void BravePrivacyHandler::AddLoadTimeData(content::WebUIDataSource* data_source,
       "isLocalhostAccessFeatureEnabled",
       base::FeatureList::IsEnabled(
           brave_shields::features::kBraveLocalhostAccessPermission));
+  data_source->AddBoolean("isBraveAIChatFeatureEnabled",
```

Comment:
this is not just checking the feature, it's checking the feature and pref so isn't it just `isAIChatEnabled`? Adding `Brave` here also makes it harder to find this by searching so it's best to stick with consistent naming

Reply:
or if you only mean to check the feature flag here then you should check that directly and not check the pref

Reply:
renamed in 52a60130f4680ac6b22955206ff08c0ad47d127a

Reply:
https://github.com/brave/brave-core/pull/26082/commits/5ab1ba2cff13f2be4aef37dd3acd3589fcaa96ec
Changed again after introducing the feature flag, it should be checking both IsAIChatEnabled and the feature flag for Open Leo from Search.